discopop_explorer/PETGraphX.py

```diff
@@ -1154,7 +1152,7 @@ def unused_check_alias(self, s: NodeID, t: NodeID, d: Dependency, root_loop: Nod
         parent_func_source = self.get_parent_function(self.node_at(t))
 
         res = False
-        d_var_name_str = cast(str, str(d.var_name))
+        d_var_name_str = str(d.var_name)
```

Comment:
this seems weird... 
`str(None)` becomes "None" string in python, and I do not think this is what we want....?

Reply:
fixed